modules/registry-configuring-storage-baremetal.adoc

```diff
@@ -5,18 +5,42 @@
 // * registry/configuring_registry_storage-baremetal
 // * installing/installing_ibm_z/installing-ibm-z.adoc
 
+ifeval::["{context}" == "installing-ibm-z"]
+:ibm-z:
+endif::[]
+ifeval::["{context}" == "installing-ibm-power"]
+:ibm-power:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-ibm-power"]
+:ibm-power:
+endif::[]
+
 [id="registry-configuring-storage-baremetal_{context}"]
+ifndef::ibm-z,ibm-power[]
 = Configuring registry storage for bare metal
+endif::ibm-z,ibm-power[]
+ifdef::ibm-z[]
+= Configuring registry storage for IBM Z
+endif::ibm-z[]
+ifdef::ibm-power[]
+= Configuring registry storage for IBM Power
+endif::ibm-power[]
 
 As a cluster administrator, following installation you must configure your
 registry to use storage.
 
 .Prerequisites
 
 * Cluster administrator permissions.
-* A cluster on bare metal.
-* Provision persistent storage
-for your cluster, such as Red Hat OpenShift Container Storage. To deploy a private image registry, your storage must provide
+* A cluster on
+ifndef::ibm-z,ibm-power[bare metal]
+ifdef::ibm-z[IBM Z]
+ifdef::ibm-power[IBM Power]
+.
+* Provision persistent storage for your cluster, such as
+ifndef::ibm-z[Red Hat OpenShift Container Storage]
+ifdef::ibm-z[NFS]
```

Comment:
Put the period after each of these two as well. This is the only bullet in this list without a period.

Reply:
Done. It's not actually a bullet, it's the first sentence of a bullet.